starboard/linux/shared/test_filters.py

```diff
@@ -34,14 +34,20 @@
         'SbSocketAddressTypes/SbSocketGetInterfaceAddressTest.SunnyDaySourceNotLoopback/type_ipv6',
     ],
     'player_filter_tests': [test_filter.FILTER_ALL],
-    'net_unittests': {test_filter.FILTER_ALL},
+    'net_unittests': [test_filter.FILTER_ALL],
 }
 
 _FILTERED_TESTS = {
     'nplb': [
         # TODO(b/286249595): This test crashes when coverage is enabled.
         'SbMemoryMapTest.CanChangeMemoryProtection'
     ],
+    'net_unittests': [
+        # TODO: b/331437821 - These pass locally and fail on GitHub.
```

Comment:
I have seen some other test cases that pass locally and fail on GitHub, like TCPClientSocketTest.BindLoopbackToLoopback and PathBuilderKeyRolloverTest.TestReturnsPartialPathEndedByLoopChecker, do you think this is a blocking issue?

Also why filtering it here? Is this only happening in Linux?

Reply:
I think investigating the failures blocks the cobalt 25 release but not the merge to trunk.

I'm not sure if it's happening on other platforms or not, I was going to wait until we're looking at them to evaluate and possible disable it via macros instead (it may work on other platforms as file tests are pretty platform-specific)